src/compiler/scala/tools/nsc/backend/jvm/opt/Inliner.scala

```diff
@@ -142,6 +142,7 @@ class Inliner[BT <: BTypes](val btypes: BT) {
           elided += r
         else
           result += r
+        ()
```

Comment:
OK, thanks - I hope it's not too common in user code that the bugfix unshadows an invalid LUB.